pydbgen/pydbgen.py

```diff
@@ -45,266 +69,151 @@ def license_plate(self,seed=None,style=None):
         If style is not specified by user, a random style is chosen at runtime
         seed: Currently not used. Uses seed from the pydb class if chosen by user
         """
-        import random
-        from random import randint,choice
         random.seed(self.seed)
 
-        if style==None:
-            style = choice([1,2,3])
-        
-        if style==1:
-            result = str(randint(1,9))

```

Comment:
String appends are slow since strings are immutable in Python. For small datasets, this wouldn't matter, but it will immediately slow down attempts to generate larger ones.